addon/components/o-s-s/slider.ts

```diff
@@ -25,7 +26,8 @@ export default class SliderComponent extends Component<SliderComponentArgs> {
   sliderOptions = {
     min: this.args.min ?? 0,
     max: this.args.max ?? 100,
-    step: this.args.step ?? 1
+    step: this.args.step ?? 1,
+    autocomplete: this.args.autocomplete ?? 'on'
```

Comment:
Same question as earlier, this one might make more sense to leave "on" though. Are we sure we want to default to this value ?